src/components/ColumnLeft/CredentialSetItem/CredentialSetItem.js

```diff
@@ -124,17 +171,102 @@ export default function CredentialSetItem({
             <div
               className={`col-md-6 text-success ${
                 new Date(
-                  `${d.substr(0, 4)}-${d.substr(4, 2)}-${d.substr(6, 2)}`
+                  `${expiry.substr(0, 4)}-${expiry.substr(
```

Comment:
mm... i know this has been there but I don't like using substr when it comes to dates, they are not strings. Would be great if we could create some helper functions or something so we can do ike

`today.subtract(20, days)` and give ISO format or unix (my preference)